lib/utils/import/evernote/enml-to-markdown.js

```diff
@@ -1,7 +1,11 @@
 import TurndownService from 'turndown';
-import { get, identity } from 'lodash';
+import { identity, startsWith } from 'lodash';
 
-const mediaPlaceholderFor = node => `(${node.getAttribute('type')})`;
+const mediaPlaceholderFor = node => {
+  const alt = node.getAttribute('alt');
+  const type = node.getAttribute('type') || 'media';
+  return alt ? `${alt}` : `(${type})`;
```

Comment:
How about we return the type and the alt text if it is available, instead of one or the other?

```
(image/png)
A picture of a beautiful flower
```

Reply:
You're right!